staging/src/k8s.io/dynamic-resource-allocation/api/doc.go

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
https://github.com/kubernetes/kubernetes/pull/127277/files#r1753074927

I guess this is auto generated so this isn't so bad.